test/crud_tests.py

```diff
@@ -140,7 +140,7 @@ def _test_database_crud(self, is_name_based):
         # delete database.
         client.DeleteDatabase(self.GetDatabaseLink(created_db, is_name_based))
         # read database after deletion
-        self.__AssertHTTPFailureWithStatus(404,
+        self.__AssertHTTPFailureWithStatus(http_constants.StatusCodes.NOT_FOUND,
```

Comment:
can you "import http_constants.StatusCodes as StatusCodes" and replace references to http_constants.StatusCodes with just StatusCodes ? here and everywhere else

Reply:
Similarly for SubStatusCodes as well